pkg/build/openbsd.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2018 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package build
+
+import (
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"time"
+
+	"github.com/google/syzkaller/pkg/osutil"
+)
+
+type openbsd struct{}
+
+func (ctx openbsd) build(targetArch, vmType, kernelDir, outputDir, compiler, userspaceDir,
+	cmdlineFile, sysctlFile string, config []byte) error {
+	const kernelName = "KCOV"
+	if err := ctx.configure(targetArch, kernelDir, kernelName); err != nil {
+		return err
+	}
+
+	compileDir := fmt.Sprintf("%v/sys/arch/%v/compile/%v", kernelDir, targetArch, kernelName)
+	if err := ctx.make(compileDir, "all"); err != nil {
+		return err
+	}
+
+	for src, dst := range map[string]string{
+		"obj/bsd": "kernel",
```

Comment:
Also copy bsd.gdb to obj/bsd.gdb (sys/targets/targets.go already knows about this file).